tree/master/cloud/src/Helper Utilities/Microsoft.Legal.MatterCenter.HelperUtilities/Microsoft.Legal.MatterCenter.UpdateAppConfig/UpdateConfig.cs

```diff
@@ -332,7 +332,13 @@ public static bool UpdateXML(Dictionary<string, string> excelInput)
                                         {
                                             case 2:
                                                 appDomain.InnerText = excelInput[ConstantStrings.AZURE_UI_SITE_URL];
-                                                break;                                            
+                                                break;
+                                            case 3:
+                                                if (appDomain.InnerText.ToLower().Contains("sharepoint"))

```

Comment:
It is needed as the AppDomains are different for the Outlook app vs the Office App.